.github/workflows/dashboard-format-v1.yaml

```diff
@@ -1,17 +1,17 @@
-name: KEEP Token Dashboard Code Format Checks
+name: KEEP Token Dashboard Code Format Checks v1
```

Comment:
Thoughts about naming the files: `dashboard-v1*.yaml` instead of `dashboard-*-v1.yaml`, i.e.:
```
dashboard-v1-format.yaml
dashboard-v1-mainnet.yaml
dashboard-v1-testnet.yaml
```

Reply:
0f607aa